src/WebJobs.Script/Environment/EnvironmentExtensions.cs

```diff
@@ -219,15 +219,31 @@ public static bool IsConsumptionSku(this IEnvironment environment)
         /// <returns><see cref="true"/> if running in the FlexConsumption Sku, false otherwise.</returns>
         public static bool IsFlexConsumptionSku(this IEnvironment environment)
         {
-            string value = environment.GetEnvironmentVariable(AzureWebsiteSku);
-            if (string.Equals(value, ScriptConstants.FlexConsumptionSku, StringComparison.OrdinalIgnoreCase))
+            string sku = environment.GetEnvironmentVariable(WebsiteSkuName);
+
+            // Currently set the sku to Dynamic in FPS
+            // TODO: Remove this once FPS sets the value to FlexConsumption
+            if (string.Equals(sku, ScriptConstants.DynamicSku, StringComparison.OrdinalIgnoreCase))
```

Comment:
This doesn't look right to me - if the sku is set to Dynamic we don't want to return true here. We only want to return true when it's "FlexConsumption". I get that you're saying that there's currently some buggy/incorrect FPS behavior that is setting this to Dynamic for Flex, but we shouldn't code to that here.

Reply:
Yes, it is a little confusing. I could just replace that environment variable entirely with a new environment variable (e.g. IS_FLEX or CONSUMPTION_VERSION).

And you're saying that we should hold off the change until the Antares change is already released with the correct environment variable, right?